Src/GUI/Plugins/PlotterPlugin.py

```diff
@@ -472,11 +472,11 @@ def on_load_data(self, event=None):
             basename = baselist[i]    
             filename = filelist[i]
             
-            f = NetCDFFile(filename,"r")
+            f = netCDF4.Dataset(filename,"r")
```

Comment:
When/where does this file get closed?

Reply:
This file should be closed at the end of the method.

Reply:
Yeap exactly